src/testing.py

```diff
@@ -23,6 +23,7 @@
 
 import proc
 
+SINGLE_THREADED = False
```

Comment:
Why?

Reply:
I'm guessing it becomes easier to debug, or to see the output?

Reply:
Yeah but it'll run the tests way slower!

Reply:
Sure, that's why it's false by default.

Reply:
I'll split this out into separate change.